### ISSUE TITLE
Truffle Compile (Named imports not working) : Cannot read property '2' of null

### ISSUE THREAD

**Issue by vince16** [trufflesuite/truffle#1271]
## Issue

Named imports are not working when I try to compile a contract using truffle. When I try to use statement like : 

**import "./V1__Item.sol" as V1__Item;**
 
then I get the following error : 
```
TypeError: Error parsing /Users/preetgsa/Projects/vpod/artemis/contracts/V2__Item.sol: Cannot read property '2' of null
    at /usr/local/lib/node_modules/truffle/build/webpack:/~/truffle-compile/parser.js:171:1
    at Array.map (<anonymous>)
    at Object.parseImports (/usr/local/lib/node_modules/truffle/build/webpack:/~/truffle-compile/parser.js:167:6)
    at /usr/local/lib/node_modules/truffle/build/webpack:/~/truffle-compile/profiler.js:302:1
    at /usr/local/lib/node_modules/truffle/build/webpack:/~/truffle-resolver/index.js:79:1
    at /usr/local/lib/node_modules/truffle/build/webpack:/~/truffle-resolver/~/async/internal/onlyOnce.js:12:1
    at next (/usr/local/lib/node_modules/truffle/build/webpack:/~/truffle-resolver/~/async/whilst.js:68:1)
    at /usr/local/lib/node_modules/truffle/build/webpack:/~/truffle-resolver/index.js:64:1
    at /usr/local/lib/node_modules/truffle/build/webpack:/~/truffle-resolver/fs.js:85:1
    at /usr/local/lib/node_modules/truffle/build/webpack:/~/truffle-resolver/~/async/internal/once.js:12:1
```
The same things works in remix ide and inter contract calls are working fine too.

## Steps to Reproduce

1. Write a contract and import another contract using a named import.
2. Compile the contract.

## Expected Behavior
The compilation should succeed.

## Actual Results

Compilation fails with the error mentioned above.

## Environment

* Operating System: macOS High Sierra 10.13.5
* Ethereum client: Ganache 1.1.0
* Truffle version (`4.11.14`):
* node version (`9.8.0`):
* npm version (`5.6.0`):

**Comment by gnidan** [comment]
Thanks for raising this.

I don't suppose you could see if this is still a problem in Truffle v5?

Install by doing:
```
npm uninstall -g truffle
npm install -g truffle@next
```

Thank you!

**Comment by vince16** [comment]
Still a problem in the latest truffle. 
Version :Truffle v5.0.0-next.12

**Comment by CruzMolina** [comment]
Thanks for raising this @vince16 !

Fix has been merged into truffle@next via #1330.
